exonum/src/runtime/versioning.rs

```diff
@@ -157,32 +157,35 @@ use crate::runtime::{ArtifactId, CoreError, ExecutionError, ExecutionFail};
 /// // Requirements can be parsed from a string.
 /// let req: ArtifactReq = "some.Service@^1.3.0".parse().unwrap();
 ///
-/// let valid_artifact = ArtifactId {
-///     runtime_id: RuntimeIdentifier::Rust as _,
-///     name: "some.Service".to_owned(),
-///     version: "1.5.7".parse().unwrap(),
-/// };
+/// let valid_artifact = ArtifactId::new(
+///     RuntimeIdentifier::Rust as u32,
+///     "some.Service".to_owned(),
+///     "1.5.7".parse().unwrap(),
+/// ).unwrap();
```

Comment:
Nit: Maybe, it could make sense to introduce the `unchecked` constructor? Alternatively, you could use a `fn main() -> Result<(), failure::Error>` wrapper for the test.

Reply:
Yeah, got idea, I've changed `unwrap`s with `?` as well.